scripts/render_DESeq2_report.parallel.R

```diff
@@ -76,15 +74,16 @@ if (!dir.exists(deglist_dir)) {dir.create(deglist_dir, recursive = TRUE)}
 
 render_report <- function(report_in, report_out, pars) {
   message("Generating report...")
-  #random_tmp <- paste0("intermediates_", stringi::stri_rand_strings(1, 10))
+  random_tmp <- file.path("/tmp",paste0("intermediates_", stringi::stri_rand_strings(1, 10)))
   rmarkdown::render(input = report_in,
                     encoding = "UTF-8",
                     output_file = report_out,
+                    #output_dir = random_tmp,
                     params = pars,
                     envir = new.env(),
-                    clean = TRUE) #,
-                    #intermediates_dir = random_tmp)
-  #system(paste0("rm -rf ", random_tmp))
+                    clean = TRUE,
+                    intermediates_dir = random_tmp)
+  system(paste0("rm -rf ", random_tmp))
 }
 
 # Determine filename prefix based on existing parameters
```

Comment:
Fix passing in params

Reply:
@mattjmeier I don't suppose you remember what this was supposed to be about? 😅

Reply:
I'm not sure what needs fixing there, no.... Haha sorry!

When I'm not on mobile I can take a look at the bigger context though, that would probably help

Reply:
nm, figured it out. It was about the scope of the params arguments